src/main/cpp/include/loader_rc/tiledb_loader_rc.h

```diff
@@ -0,0 +1,103 @@
+#include "tiledb_loader_file_base.h"
+#include "genomicsdb_config_base.h"
+#include "variant_storage_manager.h"
+#include "load_operators.h"
+#include "tiledb_utils.h"
+
+// flattened start, flattened end, name, gene, score, sample index, file index
+typedef std::tuple<int64_t, int64_t, std::string, std::string, float, int, int> cell_info;
```

Comment:
Is this the same as `transcriptomics_cell` in genomicsdb.h? Can they be refactored into a common header? Also, would prefer this to be a struct/class instead of tuple just for maintenance purposes. Adding or deleting fields may be disastrous otherwise.